src/controllers/users.js

```diff
@@ -0,0 +1,100 @@
+const User = require('../models/users');
+const Post = require('../models/posts');
+
+module.exports = {
+  // [GET] /users This route will return an array of all users.
+  index: async (req, res) => {
+    try {
+      const users = await User.find({});
+      res.status(200).json(users);
+    } catch (err) {
+      res.status(500).json(err);
+    }
+  },
+
+  // [POST] /users This route should save a new user to the server.
+  newUser: async (req, res) => {
+    try {
+      const newUser = new User(req.body);
```

Comment:
Dont do this! haha Make sure you pull the user info OFF of the body before passing it to your save.